app/views/activities/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<div>
+        <nav class="navbar">
+        <span></span>
+        <span class="login-title">Create new Transaction</span><br>
+        <%= button_to "sign out", destroy_user_session_path, class:'login-self', method: :delete %>
+    </nav>
+        <p class="notice"><%= notice %></p>
+        <p class="alert"><%= alert %></p>
+
+<div class='container input-group mb-3 contain-self'>
+<%= form_with(model: @activity) do | form|%>
+    <%= form.text_field :name, placeholder: "Name", class: 'form-control mb-1'%>
+    <%= form.text_field :amount, placeholder: "Amount", class: 'form-control mb-1'%>
+
+    <%= form.collection_select :category_ids, current_user.categories.order(:name), :id, :name, {}, { class: "form-control mt-1", multiple: true } %>
+    </div>
+    <footer type="button" class="btn btn-lg btn-block add-color-btn">
+    <%= form.submit 'Create New Transaction',class: 'btn-caty' %>
+    </footer>
+<%end%>
+</div>
+</div>
```

Comment:
- [ ]  When the user clicks on the "Back" button (<), the user navigates to the home page, Implement the back button.

- [ ] As it is shown in the recorded video when you select the added category from the list to create the transaction it is working after refreshing the page or reloading the page. So, please make it easy and quick when you select the category so that you don't have to reload/refresh the page.

![image](https://user-images.githubusercontent.com/55635977/158620288-6b4da0e9-ebce-46bc-905d-25bf44ec05ff.png)